glslc/README.asciidoc

```diff
@@ -12,6 +12,7 @@
 ----
 glslc [-c|-S|-E]
       [-x glsl] [-std=standard]
+      [-x hlsl]
```

Comment:
Yeah, I wasn't sure.  :-0
Next commit tests that -std is ignored for HLSL, and documents it.